src/deprecations.jl

```diff
@@ -379,4 +379,15 @@ until(s, f, model) = !f(model, s)
 function schedule(model::ABM, scheduler)
     @warn "`schedule(model::ABM, scheduler)` deprecated in favor of `scheduler(model)`." maxlog=1
     Iterators.filter(id -> id in allids(model), scheduler(model))
-end
\ No newline at end of file
+end
+
+# Do the standard extensions for `_exact` as in space API
+function nearby_ids_exact(agent::AbstractAgent, model::ABM, r = 1)
+    @warn "`nearby_ids_exact` is deprecated in favor of `nearby_ids(...; search=:exact)`." maxlog=1
+    all = nearby_ids_exact(agent.pos, model, r)
+    Iterators.filter(i -> i ≠ agent.id, all)
+end
+function nearby_agents_exact(a, model, r=1)
+    @warn "`nearby_agents_exact` is deprecated in favor of `nearby_agents(...; search=:exact)`." maxlog=1
+    return (model[id] for id in nearby_ids_exact(a, model, r))
```

Comment:
Shouldn't this source code just call the `nearby_ids(...; search = :exact)` instead of what it does now?

Reply:
yes, did it